src/tests/v1/test_production_locations.py

```
@@ -1,4 +1,5 @@
 import requests
+from dateutil.parser import parse
```

Ruff (0.12.2)

`dateutil.parser.parse` imported but unused

Remove unused import: `dateutil.parser.parse`

(F401)